srcpkgs/KataGo-OpenCL/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'KataGo-OpenCL'
+pkgname=KataGo-OpenCL
+version=1.11.0	
+revision=1
+#archs="i686 x86_64"
+wrksrc="KataGo-${version}"
+#create_wrksrc=yes
+build_style=cmake
+#configure_args=""
+make_build_args="-DUSE_BACKEND=OPENCL"
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends=""
+makedepends="mesa-opencl"
```

Comment:
makedepends="ocl-icd-devel zlib-devel libzip-devel", so ocl-icd is used to find right opencl implementation